classes/book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+
+  def add_rental(rental)
+    rentals.push(rental)
+  end
+end
```

Comment:
- [ ] Kindly make the `rentals` an instance variable (`@rentals`). _**Instance variables are used to store data that is specific to an object and can be accessed and modified through methods defined within the class.**_ Kindly implement this method in the initialize method of the `Book` class. 👍